API-Spec.md

```diff
@@ -0,0 +1,104 @@
+# ToDo API
+---
+This is a blueprint of the ToDo API. The API showcases POST, GET, PUT, DEL
+
+## Endpoints
+---
+
+### Create ToDo's [POST]
+Create a Todo
+
+URL: `POST/ToDos/`
+
+Request Body:
+```json
+{
+    "name": String
+    "dueDate": date
+    "status": bool
+}
+```
+Response Body:
+
+Response Code: `201 CREATED`
+
+```json
+{   
+    "id": int
+    "name": String
+    "dueDate": date
+    "status": bool
+}
+```
+
+### Fetch all ToDo's [GET]
+Get every ToDo
+
+URL: `GET/ToDos/`
```

Comment:
generally do not include the trailing slash `/` at the end of a URL as that indicates something should come after it.

Also doing that in code can break certain HTTP Clients